packages/common/src/dictionary/types.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+export type MetaData = {
```

Comment:
I'm wondering if this is the right place for MetaData Definition. It is used by dictionary but doesn't just describe dictionary. it is generic for a query.